src/services/ethereum/utils.ts

```diff
@@ -41,7 +43,18 @@ export async function fetchNetworkGasPrice(chainId: number): Promise<ethers.BigN
 }
 
 async function fetchTransactionMetadata(transaction: IScheduled): Promise<ITransactionMetadata> {
-  const provider = ethers.getDefaultProvider(ethers.providers.getNetwork(transaction.chainId));
+  let provider: ethers.providers.BaseProvider;
```

Comment:
It's good that you found this other place where the change is needed BUT now we have the same problem again - there's 2 copies of the exact same code in 2 different places. Whenever that happens ALWAYS try to find a common place for that code.
In this case we can use `ethereum/utils.ts` to store the code. So just extract this into a new function `getProvider` and reference that here and in `transaction.ts`.
Also do a global search in the project and make sure there aren't any other places where this change is needed.